sklearn/decomposition/tests/test_factor_analysis.py

```diff
@@ -57,7 +57,7 @@ def test_factor_analysis():
         # Model Covariance
         mcov = fa.get_covariance()
         diff = np.sum(np.abs(scov - mcov)) / W.size
-        assert diff < 0.1, "Mean absolute difference is %f" % diff
+        assert diff < 0.2, "Mean absolute difference is %f" % diff
```

Comment:
When I added the `global_random_seed` fixture I got the following test failures: 
```
FAILED sklearn/decomposition/tests/test_factor_analysis.py::test_factor_analysis[16] - AssertionError: Mean absolute difference is 0.170675
FAILED sklearn/decomposition/tests/test_factor_analysis.py::test_factor_analysis[38] - AssertionError: Mean absolute difference is 0.130926
FAILED sklearn/decomposition/tests/test_factor_analysis.py::test_factor_analysis[96] - AssertionError: Mean absolute difference is 0.134258
```
By increasing the threshold the failures are fixed. `0.2` looks acceptable to me but let me know if you disagree.